.golangci.yml

```diff
@@ -4,4 +4,9 @@ run:
 linters:
   enable:
     - gofmt
-    - exportloopref
+    - megacheck
+  presets:
+    - bugs
+    - unused
+  disable: 
+    - scopelint # obsoleted by exportloopref
```

Comment:
You removed exportloopref though?

Reply:
it's part of the `bugs` preset, see diff in the PR comment

Reply:
Ah, nice -- I had missed that in the clutter

Reply:
Reworded PR comment to make this clearer 👍